src/pages/our-work.js

```diff
@@ -0,0 +1,188 @@
+import React, { Fragment } from 'react'
+import Helmet from 'react-helmet'
+import { StaticQuery, graphql } from 'gatsby'
+import { Padding } from 'styled-components-spacing'
+
+import Layout from '../components/layout'
+import GreyBackground from '../components/GreyBackgroundWithoutOffset'
+import { Grid, Row, Col } from '../components/grid'
+import { H1, H3 } from '../components/Typography'
+import Hr from '../components/Common/Hr'
+import CaseStudy from '../components/OurWork/CaseStudy'
+
+const SubHeading = H3.withComponent('h2')
```

Comment:
according to the [docs](https://www.styled-components.com/docs/api#withcomponent), it seems this method is being deprecated, but we can use the `as='h2'` prop instead (it's not a change necessary for the MR to be accepted, but I thought I should mention :) )

Reply:
Ah that's great that you pointed this out. I had heard this at some point, but completely forgotten about it.

Do you wanna raise a ticket, for correcting this throughout the codebase? Maybe stick it in 'Ready for Engineering', since we seem to be using that column for Tech Debt atm.

Reply:
Sure, I'll create it right now ;)

Reply:
done https://trello.com/c/bVAB5XQc